fastlane/jetpack_metadata/android/en-US/changelogs/1314.txt

```diff
@@ -0,0 +1,4 @@
+* [*] [internal] Bump Aztec version to `1.6.3` [https://github.com/wordpress-mobile/WordPress-Android/pull/17852]
+* [**] [Jetpack-only] Blogging Prompts: add the ability to answer previous prompts, disable prompts, and other minor enhancements. [https://github.com/wordpress-mobile/WordPress-Android/pull/17882]
+* [*] Reader: Removes the option to block and report reader users for self-hosted sites [https://github.com/wordpress-mobile/WordPress-Android/pull/17866]
```

Comment:
🤔 I'm surprised by seeing the raw release notes here as well as in the previous version (the `1311.txt` file)

I run `git log -p fastlane/metadata/android/en-US/changelogs` on `trunk` and saw that `1311.txt` is the first instance (at least as far back as 19.2, after which I stopped checking) where the notes have been raw instead of editorialized.

Am I missing something or has a path got mangled at some point while touching automation code?

Regardless, the notes on Google Play are editorialized, so all good from a user's perspective

<img width="974" alt="image" src="https://user-images.githubusercontent.com/1218433/219992769-a1a1a3cf-74ec-463c-ac12-10113d41f6b6.png">

Reply:
Yeah something weird happened here, so I did it by hand. Probably just a hiccup from running it on my new machine for the first time, so I'll keep an eye out for it next time